packages/nodes-base/nodes/NocoDB/GenericFunctions.ts

```diff
@@ -55,8 +69,10 @@ export async function apiRequest(this: IHookFunctions | IExecuteFunctions | ILoa
 		delete options.body;
 	}
 
+	const credentialType = authenticationMethod === 'nocoDb' ? 'nocoDb' : 'nocoDbApiToken';
+
```

Comment:
Is this necessary? Is the value of `credentialType` not simply again the value of `authenticationMethod` afterward?

Reply:
It is, Good catch. Have now changed it.